internal/uvm/cpugroups.go

```diff
@@ -2,54 +2,30 @@ package uvm
 
 import (
 	"context"
-	"errors"
-	"fmt"
 
 	"github.com/Microsoft/hcsshim/internal/cpugroup"
 	hcsschema "github.com/Microsoft/hcsshim/internal/schema2"
 )
 
 // ReleaseCPUGroup unsets the cpugroup from the VM
+// It does this by setting the VM's cpugroup to the null group ID of 00000000-0000-0000-0000-000000000000
+//
+// Since a VM must be moved to the null group before potentially being added to a different
+// cpugroup, that means there may be a segment of time that the VM's cpu usage runs unrestricted.
 func (uvm *UtilityVM) ReleaseCPUGroup(ctx context.Context) error {
-	groupID := uvm.cpuGroupID
-	if groupID == "" {
+	if uvm.cpuGroupID == "" {
```

Comment:
This value is never set except for in this function, so we'll always return nil.